src/order-history/service.js

```diff
@@ -1,36 +1,25 @@
-/* eslint-disable no-console */
-// NOTE: console logs are intentionally added for REV-2577
-
 import { getAuthenticatedHttpClient, getAuthenticatedUser } from '@edx/frontend-platform/auth';
 import { getConfig } from '@edx/frontend-platform';
 
-const ECOMMERCE_RECEIPT_BASE_URL = `${process.env.RECEIPT_URL}`;
-
-const decimalishMatcher = /^([0-9]*[.,]*)+[0-9]*$/;
+const { ORDER_HISTORY_URL, RECEIPT_URL } = getConfig();
 
-function isNotDecimalish(num) {
-  if (Number.isNaN(num)) {
-    return true;
-  }
-
-  return !decimalishMatcher.test(num);
-}
+const ECOMMERCE_API_BASE_URL = `${ORDER_HISTORY_URL}`;
+const ECOMMERCE_RECEIPT_BASE_URL = `${RECEIPT_URL}`;
```

Comment:
Is the purpose of assigning values to these existing config flags, instead of directly using new config flags, to keep the codebase aligned for open-source?

If so, we are not fully aligned and might break some URLs. However, this workaround would work for anyone spinning up the devstack on `2u/replatforming` branch